packages/subgraph/schema.graphql

```diff
@@ -1,10 +1,86 @@
+interface IPlugin {
+  id: ID! # PluginInstallationEntityId
+  dao: Dao!
+  pluginAddress: Bytes!
+}
+
 # Types
-type DaoPlugin @entity {
-  "OSX related data"
-  id: ID! # psp installationId
+type MultisigPlugin implements IPlugin @entity {
+  id: ID! # PluginInstallationEntityId
```

Comment:
In tokenVoting, the tests and logic are setup to use the plugin address. Might wish to verify the generation process.